src/clients/api/mutations/redeemUnderlying.ts

```diff
@@ -0,0 +1,16 @@
+export interface IRedeemUnderlyingInput {
+  tokenContract: $TSFixMe; // @TODO: use contract type once defined (Typechain?)
+  account: string | undefined;
+  amount: string;
+}
+
+export type RedeemUnderlyingOutput = void;
+
+const redeemUnderlying = async ({
+  tokenContract,
+  account,
+  amount,
+}: IRedeemUnderlyingInput): Promise<RedeemUnderlyingOutput> =>
+  tokenContract.methods.redeemUnderlying(amount).send({ from: account! });
```

Comment:
I'd avoid using the `!` here. If you want account to also be defined, I'd change its type to disallow being `undefined`.